spec/default_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe 'impala::default' do
+  context 'on Centos 6.5 x86_64' do
+    let(:chef_run) do
+      ChefSpec::SoloRunner.new(platform: 'centos', version: 6.5) do |node|
+        node.automatic['domain'] = 'example.com'
+        stub_command(/update-alternatives --display /).and_return(false)
+      end.converge(described_recipe)
+    end
+
+    it 'creates impala user' do
+      expect(chef_run).to create_user('impala')
+    end
+
+    it 'create impala group' do
```

Comment:
it is just a description, but for consistency, should be: it 'creates impala group' do